_posts/2016-08-22-CustomEntryTypes.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Customize entry types in JabRef: Video tutorial
+id: CustomEntryTypesVideoTut
+author: "[JabRef Developers](https://github.com/orgs/JabRef/teams/developers)"
+---
+
+BibTeX and BibLaTeX allow users to define their own entry types and fields.
+As a leading reference management software using Bib(La)TeX format natively,
+JabRef includes this capability.
+
+In JabRef, you can [customize entry types](http://help.jabref.org/en/CustomEntriesHelp) in just a few clicks.
+A tutorial by the *Khmer TeX Users Group* illustrates this feature.
+
+[![Tutorial: customizing an entry type in JabRef](../img/CustomEntryType-Youtube-screenshot.png)](https://www.youtube.com/watch?v=iM1pg3jAIgs " Customize Entry Types in JabRef ")
+
+_**You like it? Foster JabRef development with [Code](https://github.com/JabRef/jabref/blob/master/CONTRIBUTING.md) or [Money](https://github.com/JabRef/jabref/wiki/Donations).**_
```

Comment:
No need to put that manually, it is now generated automatically. I will merge and fix afterwards.

I will also change the date to a future date so that this video is posted a few days after our release